### ISSUE TITLE
W3C Validation

### ISSUE THREAD

**Issue by Badr-Eddine-Slaoui** [Badr-Eddine-Slaoui/Gather#36]
Validate HTML and fix errors, optimize accessibility, add missing labels or ARIA attributes.

**Priority:** High
